spec/requests/providers/check_benefits_spec.rb

```diff
@@ -218,7 +218,7 @@
       let(:application) { create(:legal_aid_application, :with_positive_benefit_check_result, :checking_applicant_details, applicant:, provider:) }
 
       context "with permissions passported" do
-        let(:provider) { create(:provider, :with_passported_permissions) }
```

Comment:
expanding this out i notice there are now two tests, doing the same thing.

Reply:
the `context`s at lines 220 and 229 can probably be removed and replaced with a single `it` example

Reply:
```
        it "displays positive DWP result content" do
          get "/providers/applications/#{application.id}/check_benefits"
          expect(response.body).to include("DWP records show that your client receives a passporting benefit")
        end
```

Reply:
👍 have replaced with single test as suggested.